util/sqlexec/utils_test.go

```diff
@@ -390,39 +386,45 @@ func (s *testUtilsSuite) TestEscapeSQL(c *C) {
 			output: "select 55.2,0.66",
 		},
 	}
-	for _, t := range tests {
-		comment := Commentf("%s", t.name)
+	for _, v := range tests {
 		r3 := new(strings.Builder)
-		r1, e1 := escapeSQL(t.input, t.params...)
-		r2, e2 := EscapeSQL(t.input, t.params...)
-		e3 := FormatSQL(r3, t.input, t.params...)
-		if t.err == "" {
-			c.Assert(e1, IsNil, comment)
-			c.Assert(string(r1), Equals, t.output, comment)
-			c.Assert(e2, IsNil, comment)
-			c.Assert(r2, Equals, t.output, comment)
-			c.Assert(e3, IsNil, comment)
-			c.Assert(r3.String(), Equals, t.output, comment)
+		r1, e1 := escapeSQL(v.input, v.params...)
+		r2, e2 := EscapeSQL(v.input, v.params...)
+		e3 := FormatSQL(r3, v.input, v.params...)
+		if v.err == "" {
+			t.Run(v.name, func(t *testing.T) {
```

Comment:
Maybe run all things instead of the assertion only. You see, our intention is to construct a separated test for every loop iteration, not for the assertion. And again, every loop iter is distinct, thus can be paralleled.

Reply:
good suggestion, learned a lot, thanks. fixed